bucket/jazz2-resurrection.json

```
@@ -0,0 +1,48 @@
+{
+    "version": "0.7.3",
+    "description": "Jazz² Resurrection is an open-source remaster of the game Jazz Jackrabbit 2",
```

It's a reimplementation, as it doesn't use any of the original code (which was never released):

```suggestion
    "description": "Open source reimplementation of the game Jazz Jackrabbit 2",
```